dataactbroker/README.md

```diff
@@ -801,6 +801,67 @@ List submissions for all agencies for which the current user is a member of. Opt
 }
 ```
 
+#### GET "/v1/list_agencies/"
+Gets all CGACS that the user has submit/certify permissions
+
+Example input:
+
+None
+
+Example output:
+
+```json
+{
+    "cgac_agency_list": [
+      {
+        "agency_name": "Sample Agency",
+        "cgac_code": "000"
+      }, ...
+    ]
+}
+```
+
+#### GET "/v1/list_all_agencies/"
+Gets all CGACS
+
+Example input:
+
+None
+
+Example output:
+
+```json
+{
+    "cgac_agency_list": [
+      {
+        "agency_name": "Sample Agency",
+        "cgac_code": "000"
+      }, ...
+    ]
+}
+```
+
+#### GET "/v1/list_sub_tier_agencies/"
+Gets all CGACS that the user has submit/certify permissions
```

Comment:
This isn't exactly accurate. I mean, it does do this, but it also gets all sub tier agencies listed under that CGAC